deployment/portable/.env.default

```
@@ -1,5 +1,6 @@
 DATABASE_HOST=xyne-db
 VESPA_HOST=vespa
+OCR_PROVIDERS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OCR_PROVIDERS key should go before the VESPA_HOST key

(UnorderedKey)